src/signals/incident-management/containers/IncidentSplitContainer/IncidentSplitContainer.js

```diff
@@ -79,16 +89,18 @@ const IncidentSplitContainer = ({ FormComponent }) => {
         text_extra,
       };
 
-      const mergedData = formData.reduce((acc, { sub_category, text, type, priority }) => {
-        const partialData = {
-          category: { sub_category },
-          priority: { priority },
-          text,
-          type: { code: type },
-        };
+      const mergedData = formData.incidents
+        .filter(issue => issue)
```

Comment:
Why do we need this here? Is incidents not an array already?

Reply:
Because the department data which is requested in another ticket will apply for all incidents.